Resources/Prototypes/_Moffstation/Entities/Objects/Devices/guardian_activators.yml

```diff
@@ -0,0 +1,8 @@
+- type: entity
+  name: flying dutchman injector
+  parent: HoloparasiteInjector
+  id: FlyingDutchmanInjector
+  description: A complex artwork of handheld machinery allowing the user to summon a flying dutchman guardian.
```

Comment:
Ghosts are just a cybersun psyop to obscure the fact that their hard light tech is that advanced, smh